packages/calypso-e2e/src/lib/blocks/pricing-table-block.ts

```diff
@@ -0,0 +1,56 @@
+import { Frame, Page } from 'playwright';
+import { BaseBlock } from '../base-block';
+
+const selectors = {
+	block: '.wp-block-coblocks-pricing-table',
+	pricing: '.wp-block-coblocks-pricing-table-item__amount',
+	gutterControl: 'div[aria-label="Editor settings"] div[aria-label="Gutter"]',
+};
+
+/**
+ * Represents the Pricing Table coblock.
+ */
+export class PricingTableBlock extends BaseBlock {
+	/**
+	 * Enters the price to the side chosen.
+	 *
+	 * @param {'left'|'right'} side Left or right side of the block.
+	 * @param {string} price Price to be entered.
+	 * @returns {Promise<void>} No return value.
+	 */
+	async enterPrice( side: 'left' | 'right', price: string | number ): Promise< void > {
+		const index = side === 'left' ? 1 : 2;
+		const priceHandler = await this.block.waitForSelector(
+			`:nth-match(${ selectors.pricing }, ${ index })`
+		);
+		await priceHandler.fill( price.toString() );
+	}
```

Comment:
nit: We can make it a bit simpler:
```suggestion
	async enterPrice( column: number, price: string | number ): Promise< void > {
		const priceHandler = await this.block.waitForSelector(
			`:nth-match(${ selectors.pricing }, ${ column })`
		);
		await priceHandler.fill( price.toString() );
	}
```

Do you think we should also make it self-validate?

Reply:
I do not think this method should validate - that is the job of the `validatePublishedContent`.

Reply:
I've updated the method to the slightly simpler one you suggested. I think the column number works better as well, due to me finding out now that it's possible to alter the number of columns. That might be a good future test.

Reply:
> I do not think this method should validate - that is the job of the `validatePublishedContent`.

I think there are two sides to this:
1. Making sure that the content has actually been entered into the block when editing, and
2. Making sure that the content is rendered correctly in the front-end.

It's possible for the content to be broken in the editor but OK in the front-end, and vice-versa (see https://github.com/Automattic/wp-calypso/issues/54276). When we make the editor methods self-validate, IMO we're kinda covering the part where we'd need to have separate tests for these methods ( e.g. "enterPrice fills the price correctly"). Having said that, I think that in this case the `fill` method's actionability checks are enough of a self-validation, so it's all good here! 😄